test/private/initializeGlobalState.m

```diff
@@ -0,0 +1,47 @@
+function orig = initializeGlobalState()
+% Initialize global state. Set various properties of the graphical root to ensure reliable output of the ACID testsuite.
+% See #542 and #552
+    fprintf('Initialize global state...\n');
+
+    %--- Define desired global state properties
+    % See http://undocumentedmatlab.com/blog/getundoc-get-undocumented-object-properties
+    new.defaultAxesColorOrder.val   =               ...
```

Comment:
You got my intention. Would you prefer that I comment out this section (and add a comment) or remove it by an additional commit?

Adding a note to the tests could be done, but I fear that `stat.issues` will be cluttered with this information. Furthermore -- what gain do we have?